packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -257,8 +257,8 @@ export const InputBaseInput = styled('input', {
 );
 
 const InputGlobalStyles = globalCss({
-  '@keyframes mui-auto-fill': { from: { display: 'block' } },
-  '@keyframes mui-auto-fill-cancel': { from: { display: 'block' } },
+  '@keyframes mui-auto-fill': {},
+  '@keyframes mui-auto-fill-cancel': {},
```

Comment:
Why not removing the `InputGlobalStyles` entirely?

Reply:
Per the history of those lines, it was required for the autofill detection logic to work. But, ok, I didn't test if it's still true.

Reply:
If this trick involving the dummy animation to expose a JS hook still works, it may be needed to fix https://github.com/mui/material-ui/pull/44135#issuecomment-2420138949